runtimes/datascience/ubi9-python-3.12/Dockerfile.cpu

```diff
@@ -246,9 +246,7 @@ RUN --mount=type=cache,target=/root/.cache/pip \
     echo "Installing softwares and packages" && \
     if [ "$TARGETARCH" = "ppc64le" ]; then \
         export PKG_CONFIG_PATH=/usr/local/lib/pkgconfig; \
-        export LD_LIBRARY_PATH=/usr/local/lib64:/usr/local/lib:/usr/lib64:/usr/lib:$LD_LIBRARY_PATH; \
-        GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1 \
-        pip install ml-dtypes && \
```

Comment:
yes, was wondering about this one ;p

Reply:
Initially grpcio was also one of the dependency in pipfile.lock, but now I see it's getting skipped for ppc64le in pylock.toml
hence removed this flag `GRPC_PYTHON_BUILD_SYSTEM_OPENSSL=1`